src/gui/editors/SongEditor.cpp

```diff
@@ -837,9 +924,14 @@ void SongEditor::zoomingChanged()
 					setPixelsPerBar( pixelsPerBar() );
 	realignTracks();
 	updateRubberband();
+	
+	m_positionLine->zoomUpdate();
 }
 
-
+void SongEditor::playbackStateChanged()
+{
+	m_positionLine->playStateChanged();
+}
```

Comment:
Since Qt doesn't support metaobject features for nested classes, you can't use the old `SLOT(playStateChanged())` syntax. It's possible to use the new `&positionLine::playStateChanged()` syntax or a lambda function, though.
@DomClark's second suggestion also makes sense if we're not going to add something else to `positionLine::playStateChanged()`.

Reply:
I connected `Song::playbackStateChanged()` to `positionLine::update()`